meinberlin/apps/budgeting/assets/BudgetingProposalList.jsx

```diff
@@ -16,6 +16,7 @@ export const BudgetingProposalList = (props) => {
     return django.interpolate(countText, [votes])
   }
   const noResults = django.gettext('Nothing to show')
+  const endSessionString = django.gettext('Are you ready? End session')
```

Comment:
this should be are you finished right? are you ready doesn't really make sense in the context